From f2e267615ca12acc2df55dcd8de5a0c0fc496cf0 Mon Sep 17 00:00:00 2001
From: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Date: Fri, 24 May 2024 15:05:09 +0200
Subject: [PATCH] arm64: dts: st: use scmi clock for ipcc1 node

Fix clock name for IPCC1 node.

Change-Id: I82d744488e9bed218d9533912c580a85b88855f0
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/382479
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -408,7 +408,7 @@
 			interrupts = <GIC_SPI 171 IRQ_TYPE_LEVEL_HIGH>,
 				     <GIC_SPI 172 IRQ_TYPE_LEVEL_HIGH>;
 			interrupt-names = "rx", "tx";
-			clocks = <&scmi_clk CK_BUS_IPCC1>;
+			clocks = <&scmi_clk CK_SCMI_IPCC1>;
 			status = "disabled";
 		};
 
